modules/optimization/doc/content/source/reporters/ParsedScalars.md

```diff
@@ -0,0 +1,21 @@
+# ParsedScalars
+
+!syntax description /Reporters/ParsedScalars
```

Comment:
```suggestion
!syntax description /Reporters/ParsedScalarReporter
```

Reply:
I would not say Scalars since you only define one scalar reporter right?
then add Reporter at the end for disambiguation